File: python/leetcode/h-index.py
class Solution(object):
    def hIndex(self, citations):
        """
        :type citations: List[int]
        :rtype: int
        """
        ans = 0
        for i, c in enumerate(sorted(citations, reverse=True)):
            if i < c:
                ans += 1
        return ans

File: java/leetcode/h-index.java
class Solution {
    public int hIndex(int[] citations) {
        int ans = 0;
        Arrays.sort(citations);
        for (int i = citations.length - 1; i >= 0; i--) {
            if (citations.length - i - 1 < citations[i]) {
                ans++;
            }
        }
        return ans;
    }
}
